.github/workflows/build-test.yml

```diff
@@ -73,8 +73,8 @@ jobs:
       - name: Build tests
         run: $GITHUB_WORKSPACE/tools/iceoryx_build_test.sh build-test
 
-      # - name: Run tests
-      #   run: |
-      #     cd $GITHUB_WORKSPACE/build
-      #     ../tools/run_all_tests.sh
+      - name: Run tests
+        run: |
+          cd $GITHUB_WORKSPACE/build
+          ../tools/run_all_tests.sh disable-timing-tests
```

Comment:
Why did you disable the timing tests? Could we please fix them instead of disabling?

Reply:
Yes we have to fix them but not in this pull request. The timing tests are constantly failing on Mac OS and we need a better concept to handle them. @mossmaurice  If you have an idea how to fix them please let me know!
Otherwise I would suggest a special Jenkins target which handles only timing tests and therefore I would keep them disabled here.